evaluation_scripts/Score_Weekly.py

```diff
@@ -140,67 +140,27 @@
 
 
 # %%
+# Week 9 addition:
+# Replaced plotting script with functions
 
 # Add Histogram of results, plots each student's guess,
-# and the actual mean value for week one.
-plt.figure(figsize=(8, 6))
-plt.hist(forecasts1, bins=120, color='blue', alpha=0.75,
-         label='Student Guesses')
-plt.plot([obs_week]*3, np.arange(0, 3, 1), color='red',
-         linestyle='-', label='Actual mean')
-plt.title('Student Guesses, week 1')
-plt.xlabel('Flow Forecast')
-plt.ylabel('Count')
-plt.legend(loc='upper left')
-# %%
+# and the actual mean value for week 1 and week 2.
+title_string1 = 'Student guesses for Week 1'
+histogram1 = pf.get_histogram(forecasts1, obs_week, title_string1)
+
+title_string2 = 'Student guesses for Week 2'
+histogram2 = pf.get_histogram(forecasts2, obs_week, title_string2)
 
-# Add Histogram of results, plots each student's guess.
-plt.figure(figsize=(8, 6))
-plt.hist(forecasts2, bins=120)
-plt.plot([obs_week]*3, np.arange(0, 3, 1), color='orange',
-         linestyle='-', label='Actual mean')
-plt.title('Student Guesses and actual mean, week 2')
-plt.xlabel('Flow Forecast')
-plt.ylabel('Count')
-plt.legend(loc='upper right')
-# %%
-# Week 6 addition:  Line plots
 # Week 1 - Obs vs Forecasts
 class_avg1 = np.mean(forecasts1)
-
-fig, ax = plt.subplots()
-ax.plot(forecasts1, '-g', label='Forecast', alpha=.8)
-plt.axhline(y=class_avg1, linestyle='dashed',
-            label='Class Avg', alpha=.8, color='red')
-plt.axhline(y=obs_week, linestyle='dotted', label='Observed',
-            alpha=.8, color='blue')
-plt.xticks(np.arange(0, 19, 1))
-ax.set(title="Week 1 Forecasts", xlabel="Students",
-       ylabel="Weekly Avg Flow [cfs]")
-ax.legend(fancybox=True, framealpha=1, shadow=True,
-          borderpad=1)
-
-fig.set_size_inches(10, 4)
-# plt.savefig('Lplot_1.png')
-plt.show()
+title_string1 = 'Week 2 Forecasts'
```

Comment:
Same comment here as above

Reply:
Thanks for your comment. The function was modified correspondingly in plot_functions.py.